addons/mrp/__manifest__.py

```diff
@@ -23,9 +23,9 @@
         'views/stock_move_views.xml',
         'views/mrp_workorder_views.xml',
         'views/mrp_workcenter_views.xml',
+        'views/mrp_bom_views.xml',
```

Comment:
Is there a reason for this line move? If not then maybe it's better to keep your commit cleaner and leave it as is?

Reply:
Yes, there is. The action mrp_bom_form_action is set in this file and the html helper of mrp production use it :
`...ducts using <a name="%(mrp_bom_form_action)d" type='action' ta...` which is located in the mrp_production_views.xml file. Then the order matters in this case, right?

Reply:
Fair enough, I didn't realize order matters and hadn't run into that yet. I learned something!